content/docs/command-reference/params/diff.md

```diff
@@ -19,21 +19,24 @@ positional arguments:
 
 ## Description
 
-This command provides a quick way to compare parameter values among experiments
-in the repository history. Requires that Git is being used to version the
-project params.
+Provides a quick way to compare parameter values among experiments in the
+repository history. Requires that Git is being used to version the project
+params.
 
 > Parameter dependencies are defined with the `-p` option in `dvc run`. See also
 > `dvc params`.
 
 Without arguments, this command compares parameters currently present in the
 <abbr>workspace</abbr> (uncommitted changes) with the latest committed version.
+This includes everything in the default parameters file (`params.yaml`) as well
+as `params` found in `dvc.yaml` files.
+
+> Note that by default, `dvc params diff` only shows parameters that were
```

Comment:
this what would expect from _diff_ , not sure we need this note

Reply:
or we can make it part of the first paragraph

Reply:
> by default, `dvc params diff` only shows parameters that were changed

I think it's a relevant note. I shortened it and moved it to the paragraph.